pkgs/development/python-modules/parameterized/default.nix

```diff
@@ -1,39 +1,50 @@
 { lib
 , buildPythonPackage
 , fetchPypi
-, glibcLocales
-, isPy3k
 , mock
 , nose
+, pytestCheckHook
+, pythonOlder
 }:
 
 buildPythonPackage rec {
   pname = "parameterized";
   version = "0.8.1";
-  disable = !isPy3k;
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "sha256-Qbv/N9YYZDD3f5ANd35btqJJKKHEb7HeaS+LUriDO1w=";
+    hash = "sha256-Qbv/N9YYZDD3f5ANd35btqJJKKHEb7HeaS+LUriDO1w=";
   };
 
-  nativeCheckInputs = [
-    nose
+  checkInputs = [
     mock
-    glibcLocales
+    nose
+    pytestCheckHook
   ];
 
-  checkPhase = ''
-    runHook preCheck
-    LC_ALL="en_US.UTF-8" nosetests -v
-    runHook postCheck
-  '';
+  pytestFlagsArray = [
+    "parameterized/test.py"
+  ];
 
-  pythonImportsCheck = [ "parameterized" ];
+  disabledTests = [
+    # Tests seem outdated
+    "test_method"
+    "test_with_docstring_0_value1"
+    "test_with_docstring_1_v_l_"
+    "testCamelCaseMethodC"
```

Comment:
Tests work if you apply a patch from Fedora: https://github.com/NixOS/nixpkgs/commit/db43841fc56a5adff997e63a56c8150a1ca5afb1